Update

robin4127 opened the pull request:
I did some changes hope you like them and can push them to userstyles.

robin4127 commented:
Hey Floris,
Was away on holiday for a while, that is why my answer is a little late. 
Regd the code being messy --> only problem I see that there are quite rough wildcard/universal selectors being used. I didn't really go through all of the code properly just added the things that bothered me haha.

florisdg commented:
> Hey Floris,
> Was away on holiday for a while, that is why my answer is a little late.
> Regd the code being messy --> only problem I see that there are quite rough wildcard/universal selectors being used. I didn't really go through all of the code properly just added the things that bothered me haha.

That's alright with me. To be fair I have been real lazy in having clean CSS code in this theme. I don't really mind because of how lightweight it is anyways, as long as it works it's fine to me. As time passes the code will get cleaner I hope ^^

I've just merged your changes to master, made two tiny changes you can see read in the review.

Thanks again for your help.

robin4127 commented:
-Removed changing the border color around thumbnails to gray (line 319). The default white border matches better with images that have a white background. --> I am with you also found that to be an issue. 

-Removed the gray color of the sliders in Pricewatch (line 364). I feel like they don't pop out enough without the color and makes the Pricewatch filters too different from it's original look. --> maybe a gradient based on the color of the buttons(#394360)  Something like:     background-image: linear-gradient(#25418e, #95a3cb);
